pkg/assembler/backends/inmem/cve.go

```diff
@@ -138,11 +140,15 @@ func (c *demoClient) Cve(ctx context.Context, filter *model.CVESpec) ([]*model.C
 		if err != nil {
 			return nil, err
 		}
-		osv, err := c.buildCveResponse(uint32(id), filter)
+		cve, err := c.buildCveResponse(uint32(id), filter)
 		if err != nil {
+			if errors.Is(err, errNotFound) {
```

Comment:
does this need to be done for builder and artifact?

Reply:
They look like this
```go
		b, err := byID[*builderStruct](id, c)
		if err != nil {
			return nil, nil
		}
		return []*model.Builder{c.convBuilder(b)}, nil
```
Calling byID() directly, so the error can be ignored. The worry is where the BuildXResposne is used to search for the node by ID. BuildXResposne could return "not found" error from byID(), or error some other way.